Lib/test/test_gettext.py

```diff
@@ -115,6 +115,23 @@
     0x62, 0x61, 0x72, 0x00,  # Message data
 ]))
 
+
+GNU_MO_DATA_BIG_ENDIAN = base64.b64encode(bytes([
```

Comment:
It is weird to use b64encode here, and then decode it back. In future we may use bytes literals.